product_docs/docs/pem/9/installing/prerequisites.mdx

```diff
@@ -111,7 +131,7 @@ To install a Postgres Enterprise Manager server on Linux, you may need to perfor
     iptables -t filter -A INPUT -p TCP --dport 8443 -j ACCEPT
     ```
 
-4. Make sure the components Postgres Enterprise Manager depends on, such as python3, libboost, openssl (1.0.2k or later), snmp++, and libcurl, are up to date: 
+5. Make sure the components Postgres Enterprise Manager depends on, such as python3, libboost, openssl (1.0.2k or later), snmp++, and libcurl, are up to date on all servers: 
```

Comment:
@edbjunwang - can you please provide below require information for each platforms?

We do use edb-python3.x, libcurl-pem, edb-snmp++ on different platforms. We may want to maintain that matrix.

Reply:
I just created a page to clarify the dependencies: https://enterprisedb.atlassian.net/wiki/spaces/PEM/pages/3835461698/PEM+dependencies+matrix, please let me know if anything is unclear, or any important is missing.

Reply:
@edbjunwang We may require the edb-python version too per platform on that page.

Reply:
Updated on https://enterprisedb.atlassian.net/wiki/spaces/PEM/pages/3835461698/PEM+dependencies+matrix